TTS/speaker_encoder/models/lstm.py

```diff
@@ -70,24 +71,32 @@ def inference(self, x):
             d = torch.nn.functional.normalize(d, p=2, dim=1)
         return d
 
-    def compute_embedding(self, x, num_frames=160, overlap=0.5):
```

Comment:
why you removed `overlap` and add `num_eval`? Do they do the same thing?

Reply:
If you use the speaker_manager it already provides a function to compute embeddings from multiple instances.

Reply:
Can you explain what `num_eval` does here?  Does it set the overlap based on the expected number of evaluations?

Reply:
That's right, basically, if we want 10 samples regardless of the number of frames we will have 10 samples. That way we guarantee an x usage of memory all the time. It is more controllable than overlapping. And the state-of-the-art papers are using so I implemented it that way.

Reply:
But then we use the same number of frames regardless of the length of the audio right?

Reply:
yeah, exactly. It's better for memory consumption control.

Reply:
but it also prevents us to compute better features for longer audio clips. Am I wrong ?

Reply:
not exactly because we randomly choose windows on the audio, so it can get all the parts of the audio.

Reply:
I think it is better to keep both options. Maybe we can separate these two into two different functions. One with overlap and one with a number of frames. I think computing with overlap is more useful especially at the inference time.

Reply:
I'm not so sure about that. During some tests in the past, I compared the EER of the old implementation (overlapping) with this new one, both with the default parameters and the EER with the new implementation was better (ie embedding was better). This can be explained because taking the window randomly overlaps in different parts of the audio. If you still want I can add the two new functions, but I prefer to use this new one in extract_embeddings.

Reply:
ok makes sense. Let's keep it as it is now.